pycbc/transforms.py

```diff
@@ -1468,6 +1468,509 @@ def from_config(cls, cp, section, outputs):
+                                    section, 'longitude-lisa', tag)})
+        else:
+            additional_opts.update({'longitude_lisa_param':
+                                    cls._default_longitude_lisa})
+        if cp.has_option("-".join([section, outputs]), 'latitude-lisa'):
+            skip_opts.append('latitude-lisa')
+            additional_opts.update({'latitude_lisa_param': cp.get_opt_tag(
+                                    section, 'latitude-lisa', tag)})
+        else:
+            additional_opts.update({'latitude_lisa_param':
+                                    cls._default_latitude_lisa})
+        if cp.has_option("-".join([section, outputs]), 'polarization-lisa'):
+            skip_opts.append('polarization-lisa')
+            additional_opts.update({'polarization_lisa_param': cp.get_opt_tag(
+                                    section, 'polarization-lisa', tag)})
+        else:
+            additional_opts.update({'polarization_lisa_param':
+                                    cls._default_polarization_lisa})
+
+        if cp.has_option("-".join([section, outputs]), 'tc-ssb'):
+            skip_opts.append('tc-ssb')
+            additional_opts.update({'tc_ssb_param': cp.get_opt_tag(
+                                    section, 'tc-ssb', tag)})
+        else:
+            additional_opts.update({'tc_ssb_param': cls._default_tc_ssb})
+        if cp.has_option("-".join([section, outputs]), 'longitude-ssb'):
+            skip_opts.append('longitude-ssb')
+            additional_opts.update({'longitude_ssb_param': cp.get_opt_tag(
+                                    section, 'longitude-ssb', tag)})
+        else:
+            additional_opts.update({'longitude_ssb_param':
+                                    cls._default_longitude_ssb})
+        if cp.has_option("-".join([section, outputs]), 'latitude-ssb'):
+            skip_opts.append('latitude-ssb')
+            additional_opts.update({'latitude_ssb_param': cp.get_opt_tag(
+                                    section, 'latitude-ssb', tag)})
+        else:
+            additional_opts.update({'latitude_ssb_param':
+                                    cls._default_latitude_ssb})
+        if cp.has_option("-".join([section, outputs]), 'polarization-ssb'):
+            skip_opts.append('polarization-ssb')
+            additional_opts.update({'polarization_ssb_param': cp.get_opt_tag(
+                                    section, 'polarization-ssb', tag)})
+        else:
+            additional_opts.update({'polarization_ssb_param':
+                                    cls._default_polarization_ssb})
+
+        return super(GEOToSSB, cls).from_config(
+            cp, section, outputs, skip_opts=skip_opts,
+            additional_opts=additional_opts
+        )
+
+
+class LISAToGEO(BaseTransform):
+    """Converts arrival time, sky localization, and polarization angle in the
+    LISA frame to the corresponding values in the geocentric frame."""
+
+    name = "lisa_to_geo"
```

Comment:
Can't you simplify this by using a dictionary of the defaults? And then in your init you don't need separate if statements, you can just iterate over a list? It's the same comment for each transform so I won't repeat it for each.